### DISCUSSION TITLE
Loading config after profile creation via mozprofile

### DISCUSSION THREAD

**Discussion by Jaammerr** [sereneblue/chameleon#563] [Q&A]
**Hello, I'm creating profiles using the mozprofile module:**

`
from mozprofile import Profile
`
`profile = Profile(profile='./test_dir', addons=["chameleon.xpi"])`

I'm not familiar enough with Firefox. Is it possible to automatically load the config for chameleon using module methods or manually using the profile directory?

**Comment by sereneblue** [comment]
Hi @Jaammerr,

I'm not familiar with mozprofile but I don't think it can be used if you want to load Chameleon's config. You may be able to programmatically load settings into Chameleon using a library like Selenium. This wiki [link](https://sereneblue.github.io/chameleon/wiki/developer-guide/) may be helpful.
